site/app/controllers/grading/ElectronicGraderController.php

```diff
@@ -517,10 +517,11 @@ public function showStatus($gradeable_id) {
         $total_who_submitted = 0;
         $peers_to_grade = 0;
         $peer_graded_components = 0;
+        $total_users_who_submitted = 0;
```

Comment:
```suggestion
        $total_users_who_submitted = [];
```

The variable shape here must be an array with key/value pairs to foreach over, otherwise line 632 would error if we hit it without re-assigning `$total_users_who_submitted`. As such we should either give a default shape that will work (`[]`) or no value at all with the expectation that it'll always be re-assigned if we need it. I don't know enough about the code to say if we will never hit 632 without re-assigning the variable, so go instead to just give a default that'll function fine there.